pkg/micro-ecc/Makefile.include

```diff
@@ -1 +1,4 @@
 INCLUDES += -I$(PKGDIRBASE)/micro-ecc
+
+# LLVM/clang can't handle the inline assembler instructions in this package
+TOOLCHAINS_BLACKLIST += llvm
```

Comment:
Same here, I do not get any errors.

`make -C tests/pkg_micro-ecc --no-print-directory BOARD=iotlab-m3 clean all TOOLCHAIN=llvm`

Do you have a failing murdock output maybe ?

Reply:
Here you go: https://ci.riot-os.org/RIOT-OS/RIOT/9398/b30bc6725d6f9ff4307275a26e754dde5904c27d/output/compile/tests/pkg_micro-ecc/samr21-xpro:llvm.txt

Reply:
Ok it fails with `samr21-xpro` but not `iotlab-m3` in docker and on my computer.

****

Reply:
Ok... I'll take the blacklisting out of this PR for now.